canonical-websites/design.ubuntu.com
# Run script

QA
--

``` bash
./run clean
./run
./run build
./run watch --watch-site
etc.
```

Check the README.md too.
